src/Umbraco.Web/Models/Mapping/ContentTypeModelMapperExtensions.cs

```diff
@@ -69,7 +69,12 @@ internal static class ContentTypeModelMapperExtensions
             foreach (var a in add)
             {
                 //TODO: Remove N+1 lookup
-                var addCt = applicationContext.Services.ContentTypeService.GetContentType(a);
+                //since we don't really know from the alias what type of content type this is,
+                //we look up by content type first and then by media type if not found.
+                //content type aliases are unique so this should be safe to do.
+                IContentTypeComposition addCt = applicationContext.Services.ContentTypeService.GetContentType(a);
+                if (addCt == null)
```

Comment:
not the prettiest, but ok :) another option which is a lot more work is to split out this part of the mapping logic between media and content types.